receiver/hostmetricsreceiver/internal/scraper/cpuscraper/cpu_scraper.go

```diff
@@ -79,5 +85,13 @@ func (s *scraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 	}
 	s.mb.RecordSystemCPULogicalCountDataPoint(now, int64(numCPU))
 
+	cpuInfos, err := s.getCPUInfo()
+	if err != nil {
+		return pmetric.NewMetrics(), scrapererror.NewPartialScrapeError(err, metricsLen)
+	}
+	for _, cInfo := range cpuInfos {
+		s.mb.RecordSystemCPUFrequencyDataPoint(now, cInfo.frequency*1000000, fmt.Sprintf("cpu%d", cInfo.processor))
```

Comment:
nit to make more readable (better yet to also move this to a `hzInAMHz` constant or add a comment)
```suggestion
		s.mb.RecordSystemCPUFrequencyDataPoint(now, cInfo.frequency*1_000_000, fmt.Sprintf("cpu%d", cInfo.processor))
```